process/block/poolsCleaner/txsPoolsCleaner.go

```diff
@@ -259,11 +263,22 @@ func (tpc *txsPoolsCleaner) cleanTxsPoolsIfNeeded() int {
 			"type", getTxTypeName(currTxInfo.txType))
 	}
 
+	numTxsRounds := len(tpc.mapTxsRounds)
+	tpc.mutMapTxsRounds.Unlock()
+
+	startTime := time.Now()
+	for hash, txStore := range hashesToRemove {
+		txStore.Remove([]byte(hash))
+	}
+	elapsedTime := time.Since(startTime)
+
 	if numTxsCleaned > 0 {
-		log.Debug("txsPoolsCleaner.cleanTxsPoolsIfNeeded", "num txs cleaned", numTxsCleaned)
+		log.Debug("txsPoolsCleaner.cleanTxsPoolsIfNeeded",
```

Comment:
do we need the same log duplicated ?

Reply:
Its not duplicated. This one display the number of txs cleaned if they were, and the above one display the number of txs which are in the whole map, and could be cleaned in the future if they will remain unprocessed. Also the elapsed time displayed here refers to the time used to remove the txs from cacher, and the one above refers to the whole time consumed by this method -> cleanTxsPoolsIfNeeded

Reply:
Its not duplicated. This one display the number of txs cleaned if they were, and the above one display the number of txs which are in the whole map, and could be cleaned in the future if they will remain unprocessed. Also the elapsed time displayed here refers to the time used to remove the txs from cacher, and the one above refers to the whole time consumed by this method -> cleanTxsPoolsIfNeeded